lib/pymedphys/tests/dicom/test_dose.py

```diff
@@ -91,3 +97,92 @@ def test_require_patient_orientation():
             else:
                 with pytest.raises(ValueError):
                     require_patient_orientation(ds, test_orient)
+
+
+@pytest.mark.pydicom
+def test_sum_doses_in_datasets():
+
+    data1 = np.array([[[30, 20], [10, 5]], [[40, 25], [15, 8]]], dtype=np.uint32) * int(
+        1e6
+    )
+    scale1 = 1e-8
+
+    data2 = np.array(
+        [[[350, 400], [450, 475]], [[300, 375], [425, 460]]], dtype=np.uint32
+    ) * int(1e6)
+    scale2 = 2e-9
+
+    expected_sum = np.ones((2, 2, 2))
```

Comment:
I'm a little lost here. It's not immediately clear to me why the above two dose grids should sum to a 2x2x2 array of ones...

Reply:
See update. I've now used test data that more intuitively shows this.